tabuckner/broskichat
# Integrate register w bulmaswatch

### Project Changed


**- What I Did**

- Integrated `Slate` Bulmaswatch
- Added register form
- Added testing for register form

**- How I Did It**

Bulmaswatch
- yarn add bulmaswatch
- in index.js added 'import 'bulmaswatch/slate/bulmaswatch.min.css';

Register Form
- Added 'Users/Register' inside of Components folder
- The code concept is similar to Header.jsx but with form logic
- Validation logic was not added

**- How to Test/Verify It**

- Once you have the app running you will see the register form on the homepage
- Open up your browsers console and fill out the form
- You will see the results of what you typed in the console
- The test simply checks there is only 1 `h1` element and to verify the submit form button works

**- An Image to Make Us Giggle (not mandatory but encouraged)**

![adgduoo](https://user-images.githubusercontent.com/38574392/44129703-8a209b54-9ffe-11e8-9348-9f7e10d14df4.jpg)
